NuGet.org.md

```diff
@@ -0,0 +1,74 @@
+﻿# Cake
```

Comment:
To no not add more noise in the root folder, I think we should place this one in the `nuspec` folder where icon etc. exists.

Reply:
done.